schemas/neutron_diagnostic/dd_neutron_diagnostic.xsd

```diff
@@ -55,6 +55,201 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="neutron_diagnostic_emissivity">
+		<xs:annotation>
+			<xs:documentation>Reconstructed emissivity</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="algorithm_type" type="identifier">
+				<xs:annotation>
+					<xs:documentation>Algorithm used to reconstruct the emissivity profile</xs:documentation>
+					<xs:appinfo>
+						<doc_identifier>neutron_diagnostic/neutron_algorithm_identifier.xml</doc_identifier>
+					</xs:appinfo>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="rho_tor_norm">
+				<xs:annotation>
+					<xs:documentation>Normalized toroidal flux coordinate. The normalizing value for rho_tor_norm, is the toroidal flux coordinate at the equilibrium boundary (LCFS or 99.x % of the LCFS in case of a fixed boundary equilibium calculation, see time_slice/boundary in the equilibrium IDS)</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+						<coordinate1>1...N</coordinate1>
+						<alternative_coordinate1>../psi_norm</alternative_coordinate1>
+						<units>1</units>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="FLT_1D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="psi_norm">
+				<xs:annotation>
+					<xs:documentation>Normalized poloidal flux coordinate for the reconstructed profiles, the flux being normalized to its value at the separatrix</xs:documentation>
```

Comment:
Do we need to store both toroidal and poloidal fluxes in this IDS?  They can be mapped between each other using the equilibrium IDS.